### ISSUE TITLE
Enable allow_discussion in object from Volto results in 'Constraint not satisfied' error

### ISSUE THREAD

**Issue by sneridagh** [plone/plone.restapi#675]
BadRequest: [{'field': 'allow_discussion', 'message': u'Constraint not satisfied', 'error': ConstraintNotSatisfied(u'True', 'allow_discussion')}]

**Comment by lukasgraf** [comment]
That looks like Volto actually sends the string `"True"` for the `allow_discussion` field (string instead of boolean, and also with an Uppercase `T` as in Python, not `true` as in JS).

That would be an issue with Volto then, can you please double-check?

**Comment by sneridagh** [comment]
@lukasgraf good catch! Yep, it seems it's sending a string I will check it out. Thanks!!
